packages/editor/web/src/RichContentEditor/EditorCommands.ts

```diff
@@ -293,6 +327,15 @@ export const createEditorCommands = (
     setTextAlignment: textAlignment =>
       setEditorState(setTextAlignment(getEditorState(), textAlignment)),
     _setSelection,
+    toggleBlockOverlay: blockKey => {
+      const blockElement = document.querySelector(`[data-offset-key="${blockKey}-0-0"]`);
+      const elementOverlay = blockElement?.querySelector(`[data-hook="componentOverlay"]`);
+      elementOverlay
```

Comment:
Something like this is probably nicer
```suggestion
      const element = elementOverlay ? blockElement?.querySelector(`[data-hook="componentOverlay"]`) : blockElement
      toggleOverlayBGColor(element)
```